### PR TITLE
docs(release): Add known dinghy bug in 2.20.0

### PR DESCRIPTION
**Pull request by akshayabd**

### CODE REVIEW

**Review by 317brian — APPROVED** [review]
LGTM

**Review by aimeeu — APPROVED** [review]
Preview looks good.

File: content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-20-0.md [hunk 1]
```diff
@@ -17,13 +17,23 @@ Armory Spinnaker 2.20.x requires Armory Halyard 1.8.3 or later.
...
+### Scheduled removal of Kubernetes V1 provider
 The Kubernetes V1 provider will be removed in Spinnaker 1.21 (Armory Spinnaker 2.21). Please see the [RFC](https://github.com/spinnaker/governance/blob/master/rfc/eol_kubernetes_v1.md) for more details.
 
 Breaking change: Kubernetes accounts with an unspecified providerVersion will now default to V2. Update your Halconfig to specify `providerVersion: v1` for any Kubernetes accounts you are currently using with the V1 provider.
 
 ## Known Issues
 
+### Pipelines as code behavior change for application spec
+
+If you use the application spec in your `dinghyfile`, skip Armory Spinnaker 2.20.0 and wait for 2.20.1.
```
Comment (Jossuecito) [inline content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-20-0.md]: The behavior is happening even if you don't have application specs, dinghy users want to skip to next release so their configuration for permissions or notifications is not overwritten.

Comment (akshayabd) [reply content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-20-0.md]: Updated to indicate to skip this version entirely if using Dinghy

**Review by Jossuecito — APPROVED** [review]
LGTM

**Pull request merged**